nats/client.go

```diff
@@ -130,11 +130,23 @@ func (c *Client) ConsumeSequences(ctx context.Context, handler SequenceHandler)
 		}
 
 		if hopsMsg.MessageId == HopsMessageId {
-			c.logger.Debugf("Skipping hops assignment message")
+			c.logger.Debugf("Skipping 'hops assignment' message")
```

Comment:
What does this message mean? What is a hops assignment message?

Reply:
Only change here is adding single quotes, so not part of this review.

For info, hops assignment is when we assign a hops file to a specific sequence of events so it's then used for all future events no matter what hops instance processes them